### PR TITLE
Badge

### PR DESCRIPTION
**Pull request by vanshitamoolchandani**
Created Badge component which include:
 Images round and square 
 Text round and square

### CODE REVIEW

File: component/Badges/index.html [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+        </div>
+    </div>
+
+    <!-- BADGE BODY -->
+
+
+    <div class="avatar-body">
+        <h2 class="heading">Badges</h2>
+        <h5 class="avatar-content">Badge are indication that users can set to show if they are <strong
+                style="font-style:italic ;">online, available or away</strong>. The badges are of four sizes
```
Comment (eakansh125) [inline component/Badges/index.html:58]: Inline styling can be avoided

File: component/Badges/styles.css [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+    width: 1.25rem;
+    height: 1.25rem;
+    border: 5px solid var(--white-color);
+}
+
+.badge-m{
+    width: 1rem;
+    height: 1rem;
+    border: 4px solid var(--white-color);
+}
```
Comment (Sahil25061999) [inline component/Badges/styles.css:84]: since the border color is same it can be added on the above line, rest of the code looks good can be merged

Comment (vanshitamoolchandani) [reply component/Badges/styles.css:84]: Done. Thank you

**Review by eakansh125 — COMMENTED** [review]
Except inline styling, everything looks good to me!

File: component/Badges/index.html (hunk 1 above)
Comment (vanshitamoolchandani) [reply component/Badges/index.html:58]: Thank you for reminding.

**Pull request merged**